package.json

```diff
@@ -46,6 +46,7 @@
     ]
   },
   "scripts": {
+    "shrinkwrap": "rm ./npm-shrinkwrap.json; rm -rf ./node_modules; npm i && npm shrinkwrap && npm outdated",
```

Comment:
When you wish to update dependencies in the future, running `npm run shrinkwrap` will update the shrinkwrap based on the ranges specified in package.json.  More info: https://docs.npmjs.com/cli/shrinkwrap